packages/neuron-wallet/src/controllers/wallets.ts

```diff
@@ -56,28 +56,12 @@ class WalletsController {
     }
   }
 
-  public static create = ({
-    name,
-    address,
-    publicKey,
-    password,
-  }: {
-    name: string
-    address: string
-    publicKey: Uint8Array
-    password: string
-  }): ChannelResponse<Wallet> => {
-    const keystore = Key.generateKey(password).getKeystoreString()
-    const wallet = WalletsController.service.create({
-      name,
-      keystore,
-      address,
-      publicKey,
-    })
-    if (wallet) {
+  public static create = (): ChannelResponse<string> => {
```

Comment:
This method doesn't describe what it does. Generating a mnemonic is not a persisting operation and it should not be called `Wallet.create`. No wallet is created when calling this.

Consider renaming it to `generateMnemonic`. Or expose a mnemonic helper class to UI(which is responsible for mnemonic generation, validation, etc).

Reply:
Here is controller, not keystore. I just fix it because key's interface changed. So please @Keith-CY and @BaojunCZ refactor it. 

Of course they can tell me the logic, and I refactor it.